.gitignore

```diff
@@ -103,6 +103,9 @@ prime/
 *.snap-build
 *_source.tar.bz2
 .DS_Store
+gitea_amd64*.txt
+gitea_arm64*.txt
+gitea_armhf*.txt
```

Comment:
Move to `Snapcraft` header on line 97 and change to `/gitea_*.txt` to match exactly (assuming they generate in root).

Reply:
Updated